ci-operator/step-registry/code-ready/crc/e2e/test/code-ready-crc-e2e-test-commands.sh

```diff
@@ -14,7 +14,19 @@ fi
 
 mkdir -p "${HOME}"/.ssh
 BUNDLE_VERSION="$(crc version | grep -oP '^OpenShift version\s*:\s*\K\S+')"
-BUNDLE=crc_libvirt_"${BUNDLE_VERSION}".crcbundle
+ARCH=$(uname -m)
+case "${ARCH}" in
+    x86_64)
+        BUNDLE_ARCH="amd64"
+	;;
+    aarch64)
+        BUNDLE_ARCH="arm64"
```

Comment:
why not aarch64 ?

Reply:
https://twitter.com/jonmasters/status/1491613888680542214

Reply:
@gbraad This is because how `GOARCH` works https://gist.github.com/asukakenji/f15ba7e588ac42795f421b48b8aede63 have the list of arch for go lang supported one.

Reply:
This is about the BUNDLE arch, not GO binaries

Reply:
it is a simple change in our codebase to handle `if (arm64) { bundlearch="aarch64" }`